combine-plants.lua

```diff
@@ -16,32 +16,38 @@ if args.max then max = tonumber(args.max) end
 
 local stockpile = nil
 if args.stockpile then stockpile = df.building.find(tonumber(args.stockpile)) end
-
+    
```

Comment:
Oh, you mean the extra tabs?  Sure thing, I'll do it right now.  Mind if I ask why?